src/store/actions.js

```diff
@@ -242,24 +261,21 @@ async function fetchRecentTransactions({ commit }) {
   }
 }
 
-async function fetchTradeHistory({ state, commit }, { marketName, isRequestSilent }) {
-  let history;
-  commit(isRequestSilent ? 'startSilentRequest' : 'startRequest',
-    { identifier: 'fetchTradeHistory' });
-
-  try {
-    history = await dex.fetchTradeHistory(marketName);
-    if (state.tradeHistory && state.tradeHistory.apiBuckets && state.tradeHistory.marketName === marketName) {
-      history.apiBuckets = state.tradeHistory.apiBuckets;
-    } else {
-      history.apiBuckets = await dex.fetchTradesBucketed(marketName);
-    }
-    commit('setTradeHistory', history);
-    commit('endRequest', { identifier: 'fetchTradeHistory' });
-  } catch (message) {
-    alerts.networkException(message);
-    commit('failRequest', { identifier: 'fetchTradeHistory', message });
+async function fetchTradeHistory({ state, commit }, markets) {
+  let marketTradeHistories = [];
+  commit('startRequest', { identifier: 'fetchTradeHistory' });
+  for (const market of markets) {
```

Comment:
We probably want to use the ticker API to get the information for the markets table

Reply:
I did that, but I kept this on anyways since it's a more efficient process, and the new ticker API didn't have quite the same data as the tradeHistory object. So, I'm still using the tradeHistory for the table on the pair view.